src/apouey_index.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from typing import Sequence
+from src.core import info, IND_TYPES
+
+ALPHA_LOOKUP_TABLE = {
+    3: round(np.log(2) / np.log(3), 2),
+    4: round((np.log(4) - np.log(3) / np.log(2)), 2),
+    5: 0.73,
+    6: 0.66,
+    7: 0.78,
+    8: 0.75,
+    9: 0.82,
+    10: 0.81,
+}
+
+
+def get_apouey_index(categories: IND_TYPES, responses: Sequence) -> float:
+    number_of_categories = len(categories)
+    if number_of_categories < 3 or number_of_categories > 10:
+        raise ValueError(
+            "Category out of range. Please use category within range 3-10."
+        )
+
+    alpha = ALPHA_LOOKUP_TABLE[number_of_categories]
+    proportions_info = info(responses, categories)
+
+    sum_fc_alpha = 0
+
+    for category in range(1, len(categories) - 1):
```

Comment:
Here's a fragment where the algorithm fails to get the valid result (0.545 from here: https://github.com/hs3city/ordinal-scale-stats-py/issues/33). The algorithm doesn't iterate through n-1 categories but n-2 in this case. I recommend iterating through the `categories` list directly up to `:-1` object from the list.